core/kubernetes/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "Setup Kubernetes..."
+kubectl create configmap console-config --from-file=./config/openhim-console/default.json
+kubectl create configmap core-config --from-file=./config/openhim-core/development.json
+sudo chmod u+x ./k8s.sh
```

Comment:
Maybe we can change the permission of this file directly and not have to include this step as part of the setup script